src/styles/blocks/card.scss

```
@@ -0,0 +1,112 @@
+@import "../variables.scss";
+
+.card {
+  width: 200px;
+  padding: 32px $padding $padding;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-duration: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+    transition-duration: 0.3s;
```

```suggestion
  transition-duration: 0.3s;

  &:hover {
    transform: scale(1.2);
```
Also, would be great to specify the transition properties